File: super-impactor source/VB/ExtraData.vb
Imports System
Imports System.Drawing
Imports System.Runtime.CompilerServices

Namespace SuperImpactor
	Public Class ExtraData
		Public Property AppDetailObj As AppInfos

		Public Property ButtonText1 As String

		Public Property ButtonText2 As String

		Public Property DescText As String

		Public Property HeaderColor As Color

		Public Property HeaderText As String

		Public Property MainImage As Bitmap

		Public Property MinorText As String

		Public Sub New()
			MyBase.New()
			Me.HeaderColor = Color.Black
		End Sub
	End Class
End Namespace

File: C#/ExtraData.cs
using System;
using System.Drawing;
using System.Runtime.CompilerServices;

namespace SuperImpactor
{
	public class ExtraData
	{
		public AppInfos AppDetailObj
		{
			get;
			set;
		}

		public string ButtonText1
		{
			get;
			set;
		}

		public string ButtonText2
		{
			get;
			set;
		}

		public string DescText
		{
			get;
			set;
		}

		public Color HeaderColor
		{
			get;
			set;
		}

		public string HeaderText
		{
			get;
			set;
		}

		public Bitmap MainImage
		{
			get;
			set;
		}

		public string MinorText
		{
			get;
			set;
		}

		public ExtraData()
		{
			this.HeaderColor = Color.Black;
		}
	}
}